### DISCUSSION TITLE
Does this support Password protection?

### DISCUSSION THREAD

**Discussion by GhostFire90** [open-webui/open-webui#1413] [Q&A]
I wanted to host this with some models for me and my friends to mess with, but if im going to portforward it for the whole world to see, is there some sort of verification i could enable? (password, ssh key, whatever)

**Comment by ghost** [comment]
Indeed it does support password authentication, presently it's required (see #929). RBAC is also supported:

> 🔐 Role-Based Access Control (RBAC): Ensure secure access with restricted permissions; only authorized individuals can access your Ollama, and exclusive model creation/pulling rights are reserved for administrators.

You may want to check out the [documentation](https://docs.openwebui.com) and [troubleshooting](https://docs.openwebui.com/getting-started/troubleshooting#reset-admin-password). Since [HTTPS support isn't included](https://docs.openwebui.com/faq#q-why-doesnt-the-webui-include-https-support-built-in), you'll want to front it with HTTPS somehow. A popular solution is to use a reverse proxy like Apache or Nginx.
